collections/concepts/what-are-fingerprints.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "Fingerprints"
+categories: concepts
+permalink: /concepts/what-are-fingerprints/
+nav_order: 3
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# What Is Fingerprinting?
+
+Tokenizing data replaces raw values with a Token Identifier referencing sensitive data, meaning you don’t have the raw value in your database. Without the raw value, your system isn’t able to perform simple queries that your application may require - For example, what if you need to check for duplicate or existing values? Fingerprints solve this problem by providing you an identifier representing your Token’s uniqueness without needing to read the decrypted data.
+
+<span class="base-alert info">
+  <span>
+    Need access to the data? Our [Reactors](https://developers.basistheory.com/concepts/what-are-reactors/) and [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) enable you to share, compute, and analyze anything with your sensitive data without it touching your systems.
+  </span>
+</span>
+
+## How It Works
+
+Every [Token Type](https://docs.basistheory.com/#token-types) created with the same data will have the same fingerprint. Fingerprints are:
```

Comment:
Not all Token Types are fingerprinted. Should we update this to "Every fingerprinted Token Type..."? I don't think "fingerprintable" is a word, but I like that too.

Reply:
There is a "Learn which token types support fingerprinting" link below this - do you think that's still misleading?

Reply:
Yeah, the latter contradicts this statement and we can't guarantee we capture the reader's attention throughout the entire document.